src/app/pages/services/services.component.ts

```diff
@@ -129,12 +127,24 @@ export class Services implements OnInit {
     }
 
     if (rpc === 'service.stop') {
-      let confirm = this.confirmService.confirm(T('Alert'), T('Stop this service?'), T('Stop'));
-      confirm.subscribe(res => {
-        if (res) {
-          this.updateService(rpc, service);
-        }
-      })
+      if (service.title == 'iscsitarget') {
+        this.iscsiService.getGlobalSessions().subscribe(
+          (res) => {
+            const msg = res.length == 0 ? '' : T('<font color="red"> You have ') + res.length + T(' pending active iSCSI connection(s).</font><br>Stop the service?');
+            this.dialog.confirm(T('Alert'),  msg == '' ? T('Stop this service?') : msg, true, T('Stop')).subscribe(dialogRes => {
```

Comment:
"Stop this service?"
This is fine as-is, but would it be more helpful for the message to show which service will be stopped? "Stop iSCSI?", "Stop Dynamic DNS?", "Stop `service-name`?"